pkg/config/config.go

```diff
@@ -174,6 +177,8 @@ func setDefaults() {
 
 	viper.SetDefault(BlockstoreAzureTryTimeoutKey, DefaultAzureTryTimeout)
 	viper.SetDefault(BlockstoreAzureAuthMethod, DefaultAzureAuthMethod)
+	// allow using deprecated key for AWS secret key:
+	viper.RegisterAlias(BlockstoreS3CredentialsSecretAccessKeyKey, "blockstore.s3.credentials.access_secret_key")
```

Comment:
```suggestion
	viper.RegisterAlias("blockstore.s3.credentials.access_secret_key", BlockstoreS3CredentialsSecretAccessKeyKey)
```
I think the old name should be the alias, the new name should be the new key.